### ISSUE TITLE
Composer install fails when php has bundled intl

### ISSUE THREAD

**Issue by MattyBalaam** [silverstripe/silverstripe-framework#8557]
## Affected Version

4.2.1

## Description

Installing php using homebrew has a version of php with intl bundled in. The installer searches for ext-intl and fails.

## Steps to Reproduce

1. brew install php@7.1
2. try to create new silvestripe project
3. observe error: silverstripe/framework 4.2.1 requires ext-intl * -> the requested PHP extension intl is missing from your system.

<img width="985" alt="screen shot 2018-10-31 at 07 49 03" src="https://user-images.githubusercontent.com/1246923/47773286-7b594380-dce1-11e8-909a-ab8bbce83c67.png">

**Comment by MattyBalaam** [comment]
A workaround when running any composer commmands is to add the flag `--ignore-platform-reqs`

**Comment by MattyBalaam** [comment]
Ah, OK, thinking about it more, I guess the command line is using a different version of php on my Mac machine anyway.

**Comment by robbieaverill** [comment]
Yes, I think you need to manually add brew php to your system path nowadays
